xla/service/gpu/autotuning/gemm_fusion_autotuner_test.cc

```diff
@@ -1260,6 +1316,10 @@ TEST_F(GemmFusionAutotunerTest,
 }
 
 TEST_F(GemmFusionAutotunerTest, RewritesGemmFusionToCustomKernelFusion) {
+  if (std::holds_alternative<se::RocmComputeCapability>(
```

Comment:
This pattern appears quite often. Please make a peer to `GpuComputeComp` called `IsRocm` that does the check and use it everywhere instead.